test/jdk/java/foreign/trivial/TestTrivialUpcall.java

```diff
@@ -45,7 +51,7 @@ public class TestTrivialUpcall extends UpcallTestHelper {
     @Test
     public void testUpcallFailure() throws IOException, InterruptedException {
         // test to see if we catch a trivial downcall doing an upcall
-        runInNewProcess(Runner.class, true).assertStdOutContains("wrong thread state for upcall");
+        runInNewProcess(Runner.class, true).assertStdErrContains("wrong thread state for upcall");
```

Comment:
This doesn't appear to be specific to OpenJ9: Unless there was a change since Java 20, exception messages are printed to stderr by the RI as well.

Reply:
Not really. it crashed the VM in RI with an assertion failure in which case only the message only shows up in stdout`.

Reply:
Wow, that seems rather drastic behavior for the RI.

Reply:
Yes, that's why we don't think it is appropriate from JVM perspective given the problem comes from the incorrect setting in the user applications.

Reply:
See https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/635#discussion_r1272469586.